systemtest/src/test/java/io/strimzi/systemtest/backup/ColdBackupScriptIsolatedST.java

```diff
@@ -36,11 +35,13 @@
 @Tag(INTERNAL_CLIENTS_USED)
 @IsolatedSuite
 public class ColdBackupScriptIsolatedST extends AbstractST {
-
     private static final Logger LOGGER = LogManager.getLogger(ColdBackupScriptIsolatedST.class);
 
     @IsolatedTest
     void backupAndRestore(ExtensionContext context) {
+        clusterOperator.unInstall();
+        clusterOperator = clusterOperator.defaultInstallation().createInstallation().runInstallation();
```

Comment:
Why was this change needed?

CC @see-quick

Reply:
The error I had consistently on ColdBackup ST is "ZK PVC is already created", or something like that. So I just did what I found in the LogDump ST and it worked.

Reply:
I think there has been some changes recently. Looking at the new test name, I assume this runs in isolation. Maybe @see-quick can shed a light on this.

Reply:
Additionally, if only one test case will be inside `LogDumpScriptIsolatedST` that's okay and we do not need to do such a thing but practically speaking it would not an ideal...

Reply:
@see-quick this is the error when I use before all. It happens on restore step. 

```
Error from server (AlreadyExists): error when creating "STDIN": persistentvolumeclaims "data-my-cluster-1435185444-zookeeper-0" already exists
error: no matching resources found
```

[strimzi-debug.log](https://github.com/strimzi/strimzi-kafka-operator/files/8117439/strimzi-debug.log)

Reply:
Hmmm, I see...in `@BeforeAll` seems to be missing assigning the default installation to the cluster operator instance so instead of this:
```java
@BeforeAll
    void setUp() {
        clusterOperator.unInstall();
        clusterOperator.defaultInstallation().createInstallation().runInstallation();
    }
```
you should correct it to:
```java
@BeforeAll
    void setUp() {
        clusterOperator.unInstall();
        clusterOperator = clusterOperator.defaultInstallation().createInstallation().runInstallation();
    }

Reply:
ok, fix pushed